packages/ui-calendar/src/Calendar/index.tsx

```diff
@@ -90,18 +98,91 @@ class Calendar extends Component<CalendarProps, CalendarState> {
   }, {})
   constructor(props: CalendarProps) {
     super(props)
-
-    this.state = this.calculateState(
-      this.locale(),
-      this.timezone(),
-      props.currentDate
+    const dayRefs = Array.from({ length: Calendar.DAY_COUNT }, () =>
+      createRef<HTMLElement>()
     )
+
+    this.state = {
+      ...this.calculateState(this.locale(), this.timezone(), props.currentDate),
+      dayRefs
+    }
+  }
+
+  setFirstDayTabIndex = (tabIndex: number) => {
+    const firstDay = this.state.dayRefs[0]?.current?.firstChild as HTMLElement
```

Comment:
I would also add why the `firstChild` bit is needed (because you want the button children and not the table cell)

Reply:
maybe also call the variable `firstDayButton`